examples/prometheus/prometheus.yaml

```diff
@@ -612,7 +612,11 @@ objects:
         # only scrape infrastructure components
         - source_labels: [__meta_kubernetes_namespace]
           action: keep
-          regex: 'default|logging|metrics|kube-.+|openshift|openshift-.+'
+          regex: 'default|metrics|kube-.+|openshift|openshift-.+'
```

Comment:
Why was "`logging`" removed but "`metrics`" left in place?  Weren't they renamed to have an `openshift-` prefix?  Regardless, can you add a comment here to describe why the listed namespaces are there?

Reply:
My understanding is the `metrics` will result to `openshift-metrics` project which contains all the Prometheus components (server, alertmanage ...) and IIRC we still want to scrape those so why to remove it? But I would agree that adding the `openshift-` prefix would make sense (I will push update).

But I can not fully comment on why all the listed options are there. To me this seem like this `kubernetes-service-endpoints` job is a common configuration for bunch of services/projects.

Reply:
//cc @jcantrill @mfojtik @smarterclayton WDYT, shall we add the `openshift-` prefix to the list in `kubernetes-service-endpoints` job?

Reply:
It's already in the list:
'regex: 'default|logging|metrics|kube-.+|openshift|**openshift-.+'**'

Reply:
@jcantrill Good point. So if I understand it correctly, we use `openshift-.+` pattern for newer releases and we keep `logging|metrics` like pattern for older releases of OCP.

Reply:
It's there because first release or prometheus logging lived in a namespace that did not fit the desired pattern.